corehq/apps/linked_domain/view_helpers.py

```diff
@@ -264,6 +297,11 @@ def build_view_models_from_data_models(domain, apps, fixtures, reports, keywords
         if keyword_view_model:
             view_models.append(keyword_view_model)
 
+    for ucr_expression in ucr_expressions.values():
+        ucr_expression_view_model = build_ucr_expression_view_model(ucr_expression)
+        if ucr_expression_view_model:
+            view_models.append(ucr_expression_view_model)
```

Comment:
This data model is feature flag dependent right? I see that it doesn't fit gracefully into the current design which assumes all feature flag data models are at the "domain" level. That being said, there should still be a check that the feature flag is enabled prior to building/appending these view models right? Definitely an edge case, but possible.

Reply:
Yes, this data model is FF dependent. I explicitly didn't check if the FF is enabled on the upstream domain since this will only add items into the view if there are any that exist for the domain in question (which would have required the FF to be enabled at some point). I'm always conflicted about how often we should check for a feature flag - removing a domain from this flag wouldn't delete the objects from the DB, so I feel like linking these items is still a valid operation. Happy to defer to you if you feel otherwise though.

Reply:
> removing a domain from this flag wouldn't delete the objects from the DB, so I feel like linking these items is still a valid operation

This is the edge case I was thinking of. My assumption was that once a domain is removed from a feature flag, it loses access to anything related to that feature flag. That's just an assumption though, and you know this flag and its intent better than I do so deferring to you 👍🏻